src/IndexManager.php

```diff
@@ -277,7 +292,7 @@ private function forEachChunk(ObjectManager $objectManager, array $entities, $op
                     $entity,
                     $objectManager->getClassMetadata($entityClassName),
                     $this->normalizer,
-                    ['useSerializerGroup' => $this->canUseSerializerGroup($entityClassName)]
```

Comment:
I don't think so - this private method simply passes list of groups or null, from the perspective of someone constructing `SearchableEntity` this is transparent (since passing `useSerializerGroup` => `false` vs. passing `serializerGroups` => `null` has the same effect).